spaceship.zsh

```diff
@@ -370,9 +370,9 @@ spaceship_golang_version() {
 
   command -v go > /dev/null 2>&1 || return
 
-  local gofiles=(*.go(N))
+  setopt EXTENDED_GLOB LOCAL_OPTIONS
```

Comment:
Can you, please, add a descriptive comment what is this line for?
Probably with link. Would be helpful in future.